src/test/resources/regressions/issues/000039.gobra

```diff
@@ -8,7 +8,8 @@ func test() {
   x1  := 5
   x2@ := 5
 
-  ghost L:
+  // an empty block is inserted such that the assignments are not treated being part of the ghost labeled statement
+  ghost L: {}
```

Comment:
Oh, wow. You do not need to adapt anything, but good to be aware of this.

Reply:
It's unclear how we even would fix this as it seems to mostly be a problem in our head. Our mental model of a Viper label clashes with Go's labeled statement, which however only becomes an issue for ghost labels